docs/result.rst

```diff
@@ -0,0 +1,6 @@
+Result
```

Comment:
If we need to have all of these .rst files in the repo, could we put them in their own `cli` folder like on the queenbee repo?
https://github.com/ladybug-tools/queenbee/tree/master/docs/cli

Reply:
Yes, good point that would be cleaner!